apps/authentication-portal/src/main/webapp/emailOtp.jsp

```diff
@@ -150,7 +150,11 @@
                                 <%
                                     if ("true".equals(authenticationFailed)) {
                                 %>
-                                <a class="ui button secondary"
+                                <a 
+                                    class="ui button secondary" 
+                                    onclick="resendOtp()" 
+                                    tabindex="0" 
+                                    onkeypress="javascript: if (window.event.keyCode === 13) resendOtp()"
```

Comment:
should line 162 be changed to : (changing authenticate to be wrapped in single quotes)

value="<%=AuthenticationEndpointUtil.i18n(resourceBundle, 'authenticate')%>"